docs/class-reference.rst

```diff
@@ -46,21 +46,16 @@ Signals
    ``request``
        The ``HttpRequest`` in which the user was verified.
 
-Template Tags
--------------
-.. automodule:: two_factor.templatetags.two_factor
```

Comment:
Are those now documented elsewhere?

Reply:
Can we keep making those available as template tags? Users might depend on them in their (custom) views. Or is there reason to drop them?

Reply:
I see no reason not to move them - the name used to import them in templates would remain the same.

Reply:
Ah, I misread your comment. Is it really necessary to rename the template library as well as move it? It's the only template tag library this package implements and we can always change it later if we really need to.

Reply:
(sorry, i've been a bit busy this week)

My thinking was to just move `two_factor.templatetags.two_factor` to `two_factor.plugins.phonenumber.templatetags.two_factor`. That way templates won't have to be changed and users who want phone devices on their install will already be adding `two_factor.plugings.phonenumber` to the settings.py anyway. IMO, the less changes users have to make because of this PR, the fewer bug reports we'll get later.

I don't think we need to leave an empty `two_factor.templatetags.two_factor` because our template library doesn't contain anything that's not related to phone numbers (as you pointed out).

Reply:
OK, let's rename it then :smile_cat: